aws-datastore/src/androidTest/java/com/amplifyframework/datastore/DataStoreHubEventFilters.java

```diff
@@ -52,6 +52,31 @@ public static HubEventFilter publicationOf(String modelName, String modelId) {
         };
     }
 
+    /**
+     * Watches for publication (out of mutation queue) of a given model.
+     * Creates a filter that catches events from the mutation processor.
+     * Events will pass if they mention the provided model by its name and ID,
+     * and state that it has successfully been published off of the mutation queue.
+     * @param modelName Model name, e.g. "Post"
+     * @param modelId The ID of a model instance that might be published
+     * @return A filter that watches for publication of the provided model.
+     */
+    public static HubEventFilter enqueueOf(String modelName, String modelId) {
+        return event -> {
+            if (!DataStoreChannelEventName.OUTBOX_MUTATION_ENQUEUED.toString().equals(event.getName())) {
+                return false;
```

Comment:
both these conditions are hard to read. Would it be almost better to return true instead of doing `not` something and returning false as result?

Reply:
Can't return true as there are additional checks below it to run. This code is not new, and only exposed in test package. Made some changes to re-use this code instead of duplicating.